backend/app/models/User.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+
+from .Base import Base
+
+
+class User(Base):
+    __tablename__ = "users"
+    id = Column(String, primary_key=True)  # UUID
```

Comment:
```
from sqlalchemy.dialects.postgresql import UUID
import uuid

id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
```

How about this to auto-generate UUIDs when a new user is inserted into the db?

Reply:
Yep that's a stronger guarantee, let's do that instead.